test/typescript/foo.ts

```diff
@@ -0,0 +1 @@
+module.exports = 'bar';
```

Comment:
Same with this one, i am pretty sure (but not 100%, please correct me if I'm wrong), that this is not a valid typescript file?